asserts/account_key_test.go

```diff
@@ -228,12 +247,11 @@ func (aks *accountKeySuite) TestAccountKeyCheck(c *C) {
 	trustedKey := testPrivKey0
 
 	headers := map[string]interface{}{
-		"authority-id":           "canonical",
-		"account-id":             "acc-id1",
-		"public-key-id":          aks.keyid,
-		"public-key-fingerprint": aks.fp,
```

Comment:
Nice. Replacing id+fp by a strong digest definitely feels like the right direction.